File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/Java/Price.java
package object;



// Class for tag Price
public class Price {

  String Min;
  String Max;


  // default constructor
  public Price ( String Min, String Max ) {
    this.Min = Min;
    this.Max = Max;
  
  }


  public Price ( ) {   
  
this.Min = "0";
this.Max = "99999999.9999";
  
  }
      

} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/CPlus/Price.cpp
#include <iostream> 

using namespace std;

class Price {

	private:
  	string Min_;
  	string Max_;
 	
 	public: 

  	// default constructor
	Price () {
		Min_ = "0";
		Max_ = "99999999.9999";
  	
  	}
  
  	// by parameters
  	Price (		string Min_p, string Max_p){
	  
		Min_ = Min_p;
		Max_ = Max_p;
	  
  	}
};